BasicSample/Assets/Shared/Scripts/HideHandSpheresWhenNotActive.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR;
+
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    public class HideHandSpheresWhenNotActive : MonoBehaviour
+    {
+        [SerializeField, Tooltip("Hand Sphere UI")]
+        private GameObject[] handSpheres = new GameObject[2];
+
+        void Update()
+        {
+            bool leftHandSphereVisible = IsTracked(XRNode.LeftHand) ? true : false;
```

Comment:
I'm guessing the `? true : false` here is for readability, but I'm not entirely sure it's more clear than omitting it.

Sounds like a style discussion we could have sometime, but I'm not too concerned about it here.

Reply:
I honestly believe this is better than if else conditions, at least here :)

Reply:
Fair enough. My preference would just be `bool leftHandSphereVisible = IsTracked(XRNode.LeftHand);`
Less code == less to think about, or something like that. Definitely agree it's better than writing a full if/else block

Reply:
lol, my bad. OfCourse, I can get rid of the extra. I wrote this method differently initially and was thinking along those lines.

Reply:
nw, that works too!